<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormKecamatan</class>
 <widget class="QDialog" name="FormKecamatan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Kecamatan</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nama Kecamatan</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputNama"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Kabupaten</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboKabupaten"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBatal">
       <property name="text">
        <string>Batal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
